.devcontainer.json

```diff
@@ -2,6 +2,9 @@
 	"name": "Mono",
 	"image": "mcr.microsoft.com/devcontainers/universal:focal",
 	"postAttachCommand": "source ./init.sh",
+	"containerEnv": {
```

Comment:
This is the magic that defaults us to using 17 instead of 11 🤦

Reply:
in what world would I ever think that DISABLE_JAVA_BUILD => Java 17 wild